uid2_client/uid2_token_generator.py

```diff
@@ -66,15 +68,14 @@ def default_params():
 class UID2TokenGenerator:
 
     @staticmethod
-    def generate_uid2_token_v2(id_str, master_key, site_id, site_key, params = default_params(), version=2):
+    def generate_uid2_token_v2(id_str, master_key, site_id, site_key, params=default_params(), version=2):
```

Comment:
`default_params()` will only be evaluated once, is that going to cause any problems?

Reply:
got rid of this method